problem1.py

```diff
@@ -0,0 +1,14 @@
+#time - O(nlogk)
+#space- O(k) 
+import heapq
+class Solution:
+    def kthSmallest(self, matrix: List[List[int]], k: int) -> int:
+        min_heap=[]
+        for i in range(0,len(matrix)):
```

Comment:
This heap size is n, so time complexity will be nlogn, try to use a heap of size k